psychopy/layout.py

```diff
@@ -398,102 +543,197 @@ def height(self, value):
 
 
 class Position(Vector):
+    """Class representing a position vector.
+
+    This class is used to specify the location of a point within some
+    coordinate system (e.g., `(x, y)`).
+
+    Parameters
+    ----------
+    value : ArrayLike
+        Array of coordinates representing positions within a coordinate system.
+        Positions are specified in a similar manner to `~psychopy.layout.Vector`
+        as either 1xN for single vectors, and Nx2 or Nx3 for multiple positions.
+    units : str or None
+        Units which `value` has been specified in. Applicable values are
+        `'pix'`, `'deg'`, `'degFlat'`, `'degFlatPos'`, `'cm'`, `'pt'`, `'norm'`,
+        `'height'`, or `None`.
+    win : `~psychopy.visual.Window` or None
+        Window associated with this position. This value must be specified if
+        you wish to map positions to coordinate systems that require additional
+        information about the monitor the window is being displayed on.
+
+    """
     def __init__(self, value, units, win=None):
         Vector.__init__(self, value, units, win)
 
 
 class Size(Vector):
+    """Class representing a size.
+
+    Parameters
+    ----------
+    value : ArrayLike
+        Array of values representing size axis-aligned bounding box within a
+        coordinate system. Sizes are specified in a similar manner to
+        `~psychopy.layout.Vector` as either 1xN for single vectors, and Nx2 or
+        Nx3 for multiple positions.
+    units : str or None
+        Units which `value` has been specified in. Applicable values are
+        `'pix'`, `'deg'`, `'degFlat'`, `'degFlatPos'`, `'cm'`, `'pt'`, `'norm'`,
+        `'height'`, or `None`.
+    win : `~psychopy.visual.Window` or None
+        Window associated with this size object. This value must be specified if
+        you wish to map sizes to coordinate systems that require additional
+        information about the monitor the window is being displayed on.
+
+    """
     def __init__(self, value, units, win=None):
         Vector.__init__(self, value, units, win)
 
 
-class Vertices(object):
-    def __init__(self, verts, obj=None, size=None, pos=None, units=None, flip=None, anchor=None):
+class Vertices:
+    """Class representing an array of vertices.
+
+    Parameters
+    ----------
+    verts : ArrayLike
+        Array of coordinates specifying the locations of vertices.
+    obj : object or None
```

Comment:
obj is essentially an optional alternative way to specify size and pos - you can either state them explicitly or just supply an object and use its size and pos.